test/index.js

```diff
@@ -295,15 +296,10 @@ suite('functions', function() {
   });
 
   test('empty', function() {
-    eq(Z.empty(''), '');
-    eq(Z.empty('abc'), '');
-    eq(Z.empty([]), []);
-    eq(Z.empty([1, 2, 3]), []);
-    eq(Z.empty({}), {});
-    eq(Z.empty({x: 1, y: 2, z: 3}), {});
-    eq(Z.empty(Identity('abc')), Identity(''));
+    eq(Z.empty(String), '');
+    eq(Z.empty(Array), []);
+    eq(Z.empty(Object), {});
```

Comment:
I think we can have `empty` as instance method and in that case it's a good test case:
```js
eq(Z.empty(Identity('abc')), Identity(''));
```